app/channel_test.go

```diff
@@ -2509,13 +2508,12 @@ func TestIsCRTEnabledForUser(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
-		tc := tc
```

Comment:
Yep, `t.Parallel` was the real issue. When I removed it, I mechanically removed this variable capture as well. When the linter complained I just recaptured what it was complaining about instead of the complete struct in the loop iterator. 

I've updated to keep this line as suggested.